Gemfile.lock

```diff
@@ -0,0 +1,1431 @@
+      sprockets-rails
+      tilt
+    selenium-webdriver (3.142.7)
+      childprocess (>= 0.5, < 4.0)
+      rubyzip (>= 1.2.2)
+    sentry-raven (3.0.0)
+      faraday (>= 1.0)
+    sidekiq (5.2.7)
+      connection_pool (~> 2.2, >= 2.2.2)
+      rack (>= 1.5.0)
+      rack-protection (>= 1.5.0)
+      redis (>= 3.3.5, < 5)
+    sidekiq-logging-json (0.0.19)
+      sidekiq (>= 3)
+    sidekiq-statsd (2.1.0)
+      activesupport
+      sidekiq (>= 3.3.1)
+    simplecov (0.18.5)
+      docile (~> 1.1)
+      simplecov-html (~> 0.11)
+    simplecov-html (0.12.2)
+    spring (2.1.0)
+    spring-commands-rspec (1.0.4)
+      spring (>= 0.9.1)
+    sprockets (3.7.2)
+      concurrent-ruby (~> 1.0)
+      rack (> 1, < 3)
+    sprockets-rails (3.2.1)
+      actionpack (>= 4.0)
+      activesupport (>= 4.0)
+      sprockets (>= 3.0.0)
+    stackprof (0.2.15)
+    statsd-ruby (1.4.0)
+    sync (0.5.0)
+    table_print (1.5.6)
+    term-ansicolor (1.7.1)
+      tins (~> 1.0)
+    thor (0.20.3)
+    thread_safe (0.3.6)
+    tilt (2.0.10)
+    timecop (0.9.1)
+    tins (1.24.1)
+      sync
+    tzinfo (1.2.7)
+      thread_safe (~> 0.1)
+    unf (0.1.4)
+      unf_ext
+    unf_ext (0.0.7.7)
+    unicode-display_width (1.6.1)
+    unicorn (5.5.4)
+      kgio (~> 2.6)
+      raindrops (~> 0.7)
+    warden (1.2.8)
+      rack (>= 2.0.6)
+    warden-oauth2 (0.0.1)
+      warden
+    web-console (3.7.0)
+      actionview (>= 5.0)
+      activemodel (>= 5.0)
+      bindex (>= 0.4.0)
+      railties (>= 5.0)
+    webdrivers (4.2.0)
+      nokogiri (~> 1.6)
+      rubyzip (>= 1.3.0)
+      selenium-webdriver (>= 3.0, < 4.0)
+    webmock (3.8.3)
+      addressable (>= 2.3.6)
+      crack (>= 0.3.2)
+      hashdiff (>= 0.4.0, < 2.0.0)
+    webrick (1.6.0)
+    websocket-driver (0.7.1)
+      websocket-extensions (>= 0.1.0)
+    websocket-extensions (0.1.4)
+    whenever (1.0.0)
+      chronic (>= 0.6.3)
+    with_advisory_lock (4.6.0)
+      activerecord (>= 4.2)
+    xpath (3.2.0)
+      nokogiri (~> 1.8)
+    zeitwerk (2.3.0)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  aws-sdk (~> 3)
+  bunny (~> 2.15)
+  climate_control (~> 0.2)
+  colorize (~> 0.8)
+  dalli
+  database_cleaner
+  diffy (~> 3.3)
+  factory_bot_rails (~> 5.1)
+  faker
+  fuzzy_match (~> 2.1)
+  gds-api-adapters (~> 63)
+  gds-sso (~> 14.3)
+  govspeak (~> 6.5.3)
+  govuk-content-schema-test-helpers (~> 1.6)
+  govuk_app_config (~> 2.1)
+  govuk_document_types (~> 0.9.2)
+  govuk_schemas (~> 4.0)
+  govuk_sidekiq (~> 3.0)
+  govuk_test (~> 1.0)
+  hashdiff (~> 1.0.1)
+  json-schema
+  listen
+  oj (~> 3.10)
+  pact
+  pact_broker-client
+  pg (~> 1.2.3)
+  plek (~> 3.0)
+  pry
+  pry-byebug
+  pry-rails
+  rails (~> 6)
+  rspec
+  rspec-rails (~> 4.0)
+  rubocop-govuk
+  sidekiq-unique-jobs!
+  simplecov (= 0.18.5)
+  spring
+  spring-commands-rspec
+  stackprof
+  timecop
+  web-console (~> 3)
+  webmock
+  whenever (= 1.0.0)
+  with_advisory_lock (~> 4.6)
+
+BUNDLED WITH
+   1.17.2
```

Comment:
How come this commit is adding a `Gemfile.lock` file?

Reply:
I had issues with my branch build because of a dependabot bump on master, simplest solve was just to remove it and re-bundle

Reply:
I think if you rebase against master and on f1db7a271f06475a20a85aca1b1696fc9afc695e, you can run `bundle install` which will generate the latest `Gemfile.lock`, and then move on to the next commit and for each one where you get conflicts you can run `bundle install` to keep the `Gemfile.lock` file up to date. If there are conflicts, I think `bundle install` will complain and make you first delete `Gemfile.lock` before running `bundle install`.

Reply:
The "Remove Arel gem" commit now does look better, but there are too many changes in the Gemfile.lock, only arel should be changing. Can you take a look?

Reply:
This should look much better now

Reply:
Indeed it does, good stuff :+1: